README.md

```diff
@@ -1,24 +1,10 @@
-<h1 align="center"> Figura Goofy Plugin</h1>
+<h1 align="center">Figura Goofy Plugin</h1>
+<h2 align="center">The dumping ground for the Figura Goofballs</h2>
 <p align="center">
   <img alt="fabric" height="56" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/cozy/supported/fabric_vector.svg">
-  <img alt="forge" height="56" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/cozy/supported/forge_vector.svg">
   <img alt="quilt" height="56" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/cozy/supported/quilt_vector.svg">
 </p>
 
 # Links
-[discord]: https://discord.com/api/guilds/1129805506354085959/widget.png
-[modrinth]: https://img.shields.io/badge/Modrinth-1bd96a?logo=modrinth&logoColor=ffffff&labelColor=1bd96a
-[curseforge]: https://img.shields.io/badge/CurseForge-f16436?logo=curseforge&logoColor=ffffff&labelColor=f16436
-[kofi]: https://img.shields.io/badge/Ko--fi-00b9fe?logo=kofi&logoColor=ffffff&labelColor=00b9fe
-[collective]: https://img.shields.io/badge/Open%20Collective-83b3fb?logo=opencollective&logoColor=ffffff&labelColor=83b3fb
-[collective]: https://img.shields.io/badge/Open%20Collective-83b3fb?logo=opencollective&logoColor=ffffff&labelColor=83b3fb
-[maven]: https://img.shields.io/badge/Figura%20Maven-figuraMaven?logo=GitHub&labelColor=blue&color=blue
-
-
-## Maven: [![maven]](https://maven.figuramc.org/#/releases/org/figuramc/figura-common-intermediary)
-
-## Social: [ ![discord][] ](https://discord.gg/figuramc)
-
-## Download: [ ![modrinth] ](https://modrinth.com/mod/figura) [ ![curseforge][] ](https://curseforge.com/minecraft/mc-mods/figura)
-
-## Donate: [ ![collective][] ](https://opencollective.com/figura) [ ![kofi][] ](https://ko-fi.com/skyrina)
+Discord - [<img src="https://discord.com/api/guilds/1240038278212878436/widget.png">](https://discord.com/invite/7RQYvXDXDQ)  
+Download - [<img src="https://img.shields.io/badge/Modrinth-1bd96a?logo=modrinth&logoColor=ffffff&labelColor=1bd96a">](https://modrinth.com/mod/figura-goofyplugin/)
```

Comment:
link is a 404; should this link to actions until the project is approved?

Reply:
Good idea

Reply:
would a link like this work for now? (this can be reverted once modrinth is approved)
```suggestion
Download - [<img src="https://img.shields.io/github/actions/workflow/status/Figura-Goofballs/GoofyPlugin/build.yml?labelColor=1bd96a">](https://github.com/Figura-Goofballs/GoofyPlugin/actions/workflows/build.yml)
```

Reply:
yeah until modrinth link is public lets do this